### DISCUSSION TITLE
Share Korean translation

### DISCUSSION THREAD

**Discussion by keispace** [google/comprehensive-rust#110] [General]
Hello, I'm translating it into Korean to study and share it with my team members personally. (Android part in progress) It's not difficult and long, but I'm sharing it because I think it would be good to refer to if anyone needs it. Please let me know if it's a problem.

* There may be mistranslation and it is marked with the original text for the unification of terms.

https://github.com/keispace/comprehensive-rust-kr

https://keispace.github.io/comprehensive-rust-kr/

**Comment by mgeisler** [comment]
Oh wow, thanks @keispace!

I would love to integrate this translation back into the course here :smile: It's great that you kept the original English text around in the quote blocks. That should make it possible to reconcile this later with the canonical source.

My plan for translations is to use Gettext via the infrastructure I wrote here: https://github.com/rust-lang/mdBook/pull/1864. This is the best and most standard way I know of doing translations. Let me create an issue for this so we have a place to track this.

**Reply by mgeisler** [reply]
I created #115 to track the overall work on supporting translations.

@keispace please keep up the good work and keep having the English text next to the Korean translation. We might be able to convert this automatically into Gettext format later, otherwise we'll do it via lots of copy-pasting.

**Reply by mgeisler** [reply]
Please see #130 for my PR which adds an internationalization infrastructure to the course.

**Comment by mgeisler** [comment]
We've just merged #264 which allows us to publish translations as part of the normal publishing pipeline!

**Comment by mgeisler** [comment]
The translation is now published: https://google.github.io/comprehensive-rust/ko/! Thanks for the hard work on this!
